src/concrete/ml/quantization/base_quantized_op.py

```diff
@@ -865,7 +865,7 @@ class QuantizedMixingOp(QuantizedOp, is_utility=True):
     Mixing operators cannot be fused to TLUs.
     """
 
-    lsbs_to_remove: Optional[int] = None
+    lsbs_to_remove: Optional[Union[int, dict]] = None
```

Comment:
I think it would have been better to simply duplicate the AutoRounder class. Then Gemm can create 3 auto rounders for the different "keys". 

It's a bit of a hassle though, let me know what you think

Reply:
We said we would not use the auto rounder as it's not possible to use rounding during calibration.

Reply:
I'm actually saying you could have implemented a new AutoRounder class in Concrete ML that implements this functionality during calibration. Then you would have had 3 instances of it in the Gemm class

Reply:
I am not sure what this would bring? The current function is fairly simple and the only thing we need to compute and store is lsbs_to_remove. There are no specific parameters to this. Anyway I don't think this PR is the place to introduce a new auto rounder class. Let's make an issue to discuss if we want to move toward this.